### PR TITLE
Differing payscales for differing species

### PR DESCRIPTION
**Pull request by Anvilman6**
Adds variable payscales to different roundstart species to better reflect NT's exploitative practices and views. 

# General Documentation

### Intent of your Pull Request

Adds a multiplier to account paydays, varying per roundstart species, based on NT's views of them and their overall usefulness to the corporation.

**Table of adjustments:**

- Humans / Felinids[1]: **1.0x** (baseline, and cat flavored baseline)
- Moths / Preternis / Plasmamen: **0.8x** (Very useful in various research fields and tech positions, more pay for more brains.)
- Ethereals / Pods: **0.7x** (Neutral viewpoints, no strong benefits or drawbacks. Pay them and they will fill in where needed.)
- Lizards / Polys: **0.6x** (Very alien and very discomforting, but willing to work harder for less)

[1]: Not a standalone species, but I decided to note it even so, as there will certainly be Felinid players that are frightened by this PR. Donator species is _unchanged_, so fear not.

### Why is this change good for the game?
Changes the overly simplified pay systems in-game and creates a more nuanced and immersive experience for players which is more reflective of the lore. NT is a corporation focused on profits, and if it can squeeze quality labor out of non-Human species for lower wages, it certainly will.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Modifiers to account paydays per species. These should be noted on each species page, potentially in the template header; see above table for numbers.

# Changelog

:cl:  
rscadd: Varying payscales for different species, see PR table for details!
experimental: Payscale code was drafted based on TG's PR 51212 and modified to fit Yog. The modifier numbers are subject to change if enough people desire it, so be sure to give feedback if you would like.
/:cl:

### CODE REVIEW

**Comment by boba247** [comment]
Guess I'll have to start taking bribes as sec to make ends meet.

**Comment by Jarod1200** [comment]
Very based

**Comment by ThatGuyPhil** [comment]
this but felinids are paid the least

**Comment by stwordcitizen** [comment]
Good pr

**Comment by CrazyDrakkon** [comment]
To think I used to want an implemented currency. There doesn't need to be further punishment for picking non-human.

**Comment by ToasterBiome** [comment]
This is a perfect PR. I was gonna do this but never got around to it. Finally we take a stand.

**Comment by JamieD1** [comment]
Test Merging
